.github/workflows/bdd-stylecheck.yml

```
@@ -27,6 +27,8 @@ jobs:
 
       - name: Install uv
         uses: astral-sh/setup-uv@f3121ef80ee2c4902cefcba4d30c2d17aa8baea7
```

Let's specify version here as otherwise Renovate will propose PR points to the latest commit from main.
```suggestion
        uses: astral-sh/setup-uv@a02a550bdd3185dba2ebb6aa98d77047ce54ad21 # v6.2.1
```